docs/modules/performance/pages/caching-deserialized-values.adoc

```diff
@@ -35,6 +35,7 @@ hazelcast:
       backup-count: 1
 ----
 ====
+--
```

Comment:
Why is this necessary? I haven't seen it mentioned on https://github.com/hazelcast/hz-docs/blob/main/docs-tabs-library/tabs-block.js

Reply:
I was looking for reference to understand how should it be. Thanks for the link. Looking at the link and other usages, it seems that we need to add two dashes before and after [source*] block.